tests/CMakeLists.txt

```diff
@@ -104,7 +104,7 @@ endforeach()
 add_subdirectory(psi4numpy)
 add_subdirectory(python)
 add_subdirectory(json)
-add_subdirectory(pasture)
+add_subdirectory(pasture-ccsorttransqt2)
```

Comment:
Since the test itself checks if pasture is available I didn't see any reason to. The test will run and pass if pasture is not installed. If pasture is installed it sets `run_cctransort false` and expects the same results to be obtained.